dataforest/structures/cache/PathCache.py

```diff
@@ -66,9 +70,8 @@ def _get(self, process_name: str) -> Optional[Path]:
         one doesn't exist.
         """
         process_dir = self.get_process_dir(process_name)
-        if self._exists_req:
-            if not process_dir.exists():
-                return None
+        if self._exists_req and process_dir.exists():
```

Comment:
Before that, it was `if not process_dir.exists():`?

Reply:
Oooh, good catch!